lib/pkg/flow-base/lib/FlowHelpers.js

```diff
@@ -144,12 +144,28 @@ async function canFindFlow(flowPath: string): Promise<boolean> {
 }
 
 /**
- * @return The path to Flow on the user's machine. It is recommended not to cache the result of this
- *   function in case the user updates his or her preferences in Atom, in which case the return
- *   value will be stale.
+ * @return The path to Flow on the user's machine. First using the the user's config, then looking into
+ *   the node_modules for the project.
+ *   It is recommended not to cache the result of this function in case the user updates his or her 
+ *   preferences in Atom, in which case the return value will be stale.
  */
-function getPathToFlow(): string {
-  return global.vscode.workspace.getConfiguration('flow').get('pathToFlow')
+async function getPathToFlow(): Promise<string> {
+  const config = global.vscode.workspace.getConfiguration('flow').get('pathToFlow')
+  if (await canFindFlow(config)) {
+    return config
+  }
+  if (await canFindFlow(fallbackNodeModuleFlowLocation())){
+    return fallbackNodeModuleFlowLocation()
+  }
+  return config
+}
+
+/**
+ * @return The potential path to Flow on the user's machine if they are using NPM/Yarn to manage
+ * their installs of flow.
+ */
+function fallbackNodeModuleFlowLocation(): string {
+  return global.vscode.workspace.rootPath + "/node_modules/.bin/flow"
```

Comment:
I initially thought that for windows support I would need to use [path.join](https://nodejs.org/api/path.html#path_path_join_paths) but Stack Overflow says it's [not needed](http://stackoverflow.com/questions/9756567/do-you-need-to-use-path-join-in-node-js).